Release/include/cpprest/ws_msg.h

```diff
@@ -64,6 +64,12 @@ class websocket_outgoing_message
     /// This is useful when the client side wants to check whether the server is alive.
     /// </summary>
     void set_pong_message() { this->set_message_pong(); }
+
+    /// <summary>
+    /// Sets a the outgoing message to be a ping message.
```

Comment:
Copy-and-pasted typo, change 'a the' to 'the'.

Reply:
changed too